src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/formatter/LocationFormatter.kt

```diff
@@ -6,8 +6,8 @@ import org.springframework.stereotype.Component
 class LocationFormatter {
 
   fun format(value: String): String {
-    return when (value) {
-      "RECP" -> "Reception"
+    return when {
+      value.contains("RECP") -> "Reception"
```

Comment:
making this a bit less rigid - we must get a slightly different type of description back from NOMIS for movements which is prefixed with agency ID